backend/src/users/user.service.ts

```diff
@@ -26,20 +26,16 @@ export class UsersService {
         return await this.userModel.findById(id).exec()
     }
 
-    async login(@Param() params: LoginUserDto) {
-        if (!params.password || !params.email) {
-            return null
-        }
-
+    async signIn(@Param() params: LoginUserDto) {
         try {
             const user = await this.userModel.findOne({email: params.email})
```

Comment:
You already have UserDocument in `user.controller.ts` when you call:
```ts
const emailExists = await this.userModel.findOne({email: userDto.email});
```

Rename it to `user` and do password checking there as well.
So you don't query for user's existence twice per one login request.